### PR TITLE
It seems the tempdir feature is broken

### PR DESCRIPTION
**Pull request by llyfacebook**

### CODE REVIEW

File: benchmarking/platforms/platform_base.py [hunk 1]
```diff
@@ -78,9 +78,12 @@ def preprocess(self, *args, **kwargs):
     def copyFilesToPlatform(self, files, target_dir=None):
         target_dir = (self.tgt_dir if target_dir is None else target_dir)
         if isinstance(files, string_types):
-            target_file = os.path.join(target_dir, os.path.basename(files))
-            self.util.push(files, target_file)
-            return target_file
+            if not os.path.isdir(files):
```
Comment (sf-wind) [inline benchmarking/platforms/platform_base.py]: why files can be a directory?

Comment (llyfacebook) [reply benchmarking/platforms/platform_base.py]: Sometimes, we need to create a tmp directory in the run time for storing the application files.

Comment (sf-wind) [reply benchmarking/platforms/platform_base.py]: Let's discuss this use case.

File: benchmarking/platforms/platform_base.py [hunk 2]
```diff
@@ -78,9 +78,12 @@ def preprocess(self, *args, **kwargs):
...
+                target_file = os.path.join(target_dir, os.path.basename(files))
+                self.util.push(files, target_file)
+                return target_file
+            else:
+                return files
```
Comment (sf-wind) [inline benchmarking/platforms/platform_base.py]: This doesn't work with ios/android. It's better to change the push routine to support directory there.

https://github.com/facebook/FAI-PEP/blob/master/benchmarking/platforms/host/hdb.py#L23

**Pull request merged**